packages/components/src/Icon/_docs/icon.stories.scss

```diff
@@ -13,14 +13,12 @@
 .button {
   @include button-reset;
 
-  display: flex;
+  display: inline-flex;
```

Comment:
Might need to be `flex` instead of `inline-flex`? The layout in the docs have changed

Reply:
That won't make any change. The main diff is the gaps between the cards are even because the columns are now controlled with a set value as apposed to 1fr - to me its neater